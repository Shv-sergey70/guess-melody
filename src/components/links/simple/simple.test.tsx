import * as React from 'react';
import {configure, shallow} from 'enzyme';
import * as Adapter from 'enzyme-adapter-react-16';
import Simple from "./simple";

configure({adapter: new Adapter()});

describe(`SimpleLink correctly renders`, () => {
  test(`without not required props`, () => {
    const simpleLink = shallow(<Simple to={`/main`} />);

    expect(simpleLink).toMatchSnapshot();
  });

  test(`with not required props`, () => {
    const simpleLink = shallow(
        <Simple
          to={`/main`}
          className={`myClass`}
          onClick={jest.fn()}
        >
          My custom children
        </Simple>
    );

    expect(simpleLink).toMatchSnapshot();
  });
});
