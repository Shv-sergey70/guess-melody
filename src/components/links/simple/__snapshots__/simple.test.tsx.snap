// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SimpleLink correctly renders with not required props 1`] = `ShallowWrapper {}`;

exports[`SimpleLink correctly renders without not required props 1`] = `ShallowWrapper {}`;
